test/math/FloatMatrixTest.ooc

```diff
@@ -145,7 +152,7 @@ FloatMatrixTest: class extends Fixture {
 		// 2 5
 		for (x in 0 .. matrix dimensions width) {
 			for (y in 0 .. matrix dimensions height) {
-				expect(matrix[x, y], is equal to(values[x * matrix dimensions height + y]))
+				expect(matrix[x, y], is equal to(values[x * matrix dimensions height + y]) within(this tolerance))
```

Comment:
Try storing the result of `matrix[x, y]` in a variable and using that as the first argument to `expect` and see if that works better:
```ooc
(element, value) := matrix[x, y]
expect(element, is equal to(value) within(this precision)
```